inuits-169.css

```diff
@@ -294,7 +301,7 @@ margin: 20% 0px 0px 0px;
 
 .author
 background-color: rgba(0,0,0,.7);
-background-image: url('../../images/themes/inuits/inuits2-tit.png');
+background-image: url('inuits/logo.png');
```

Comment:
Should both use the http address ?

Reply:
I'd say it makes more sense to use the local image.